packages/button/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@rainbow-me/button",
+  "description": "Rainbow connector button",
+  "version": "0.0.1",
+  "files": [
+    "dist"
+  ],
+  "keywords": [],
+  "author": "Rainbow",
+  "license": "MIT",
+  "type": "module",
+  "exports": {
+    ".": "./dist/index.js",
+    "./styles.css": "./node_modules/@rainbow-me/rainbowkit/styles.css"
+  },
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "build": "node build.js",
+    "build:watch": "node build.js --watch",
+    "dev": "pnpm build:watch & pnpm typegen:watch",
+    "prepare": "pnpm build",
+    "prebuild": "pnpm typegen",
+    "typecheck": "pnpm tsc --noEmit",
+    "typegen": "tsc --emitDeclarationOnly || true",
+    "typegen:watch": "tsc --emitDeclarationOnly --watch"
+  },
+  "peerDependencies": {
+    "@rainbow-me/rainbowkit": ">=1.1.1",
```

Comment:
We'll have to keep this in mind and first deploy the RainbowKit changes and then bump up this peer dependency